File: src/jk_5/nailed/map/gameloop/instructions/InstructionUnwatchUnready.java
package jk_5.nailed.map.gameloop.instructions;

import jk_5.nailed.map.gameloop.GameThread;
import jk_5.nailed.map.gameloop.IInstruction;

/**
 * No description given
 *
 * @author jk-5
 */
public class InstructionUnwatchUnready implements IInstruction {

    @Override
    public void injectArguments(String arguments) {
    }

    @Override
    public void execute(GameThread controller) throws InterruptedException {
        controller.setWatchUnready(false);
    }
}

File: src/main/scala/jk_5/nailed/map/gameloop/instructions/InstructionUnwatchUnready.scala
package jk_5.nailed.map.gameloop.instructions

import jk_5.nailed.map.gameloop.GameThread
import jk_5.nailed.map.gameloop.IInstruction

/**
 * No description given
 *
 * @author jk-5
 */
class InstructionUnwatchUnready extends IInstruction {
  def injectArguments(arguments: String){}
  def execute(controller: GameThread) = controller.setWatchUnready(watch = false)
}
